yt_dlp/extractor/huya.py

```diff
@@ -37,6 +37,34 @@ class HuyaLiveIE(InfoExtractor):
     }]
 
     _RESOLUTION = {
+        '蓝光30M': {
+            'width': 1920,
+            'height': 1080,
+        },
+        '蓝光20M': {
+            'width': 1920,
+            'height': 1080,
+        },
+        '蓝光15M': {
+            'width': 1920,
+            'height': 1080,
+        },
+        '蓝光12M': {
+            'width': 1920,
+            'height': 1080,
+        },
+        '蓝光10M': {
+            'width': 1920,
+            'height': 1080,
+        },
+        '蓝光8M': {
+            'width': 1920,
+            'height': 1080,
+        },
+        '蓝光6M': {
+            'width': 1920,
+            'height': 1080,
+        },
         '蓝光4M': {
             'width': 1920,
             'height': 1080,
```

Comment:
Isn't it better to just strip `\d+M$` from the key before searching in dict? instead of hard coding all these for same resolution. The extracted \d+ can be directly used for bitrate too

Reply:
Indeed. Updated :)